docs/devel/update-release-docs.md

```diff
@@ -0,0 +1,145 @@
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Table of Contents
+
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+
+- [Table of Contents](#table-of-contents)
+- [Overview](#overview)
+- [Adding a new docs collection for a release](#adding-a-new-docs-collection-for-a-release)
+- [Updating docs in an existing collection](#updating-docs-in-an-existing-collection)
+  - [Updating docs on HEAD](#updating-docs-on-head)
+  - [Updating docs in release branch](#updating-docs-in-release-branch)
+  - [Updating docs in gh-pages branch](#updating-docs-in-gh-pages-branch)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
+# Overview
+
+This document explains how to update kubernetes release docs hosted at http://kubernetes.io/docs/.
+
+http://kubernetes.io is served using the [gh-pages
+branch](https://github.com/kubernetes/kubernetes/tree/gh-pages) of kubernetes repo on github.
+Updating docs in that branch will update kubernetes.io
+
+There are 2 scenarios which require updating docs:
+* Adding a new docs collection for a release.
+* Updating docs in an existing collection.
+
+# Adding a new docs collection for a release
+
+Whenever a new release series (release-X.Y) is cut from master, we push the
+corresponding set of docs to http://kubernetes.io/vX.Y/docs. The steps are as follows:
+
+* Create a _vX.Y folder in gh-pages branch.
+* Create a new _includes/nav_vX.Y.html file with the navigation menu. This can
+  be a copy of _includes/nav_vX.Y-1.html with links to new docs added and links
+  to deleted docs removed. Update [_layouts/docwithnav.html]
+  (https://github.com/kubernetes/kubernetes/blob/gh-pages/_layouts/docwithnav.html)
+  to include this new navigation html file. Example PR: [#16143](https://github.com/kubernetes/kubernetes/pull/16143).
+* [Pull docs from release branch](#updating-docs-in-gh-pages-branch) in _vX.Y
+  folder.
+
+Once these changes have been submitted, you should be able to reach the docs at
+http://kubernetes.io/vX.Y/docs/ where you can test them.
+
+To make X.Y the default version of docs:
+
+* Update [_config.yml](https://github.com/kubernetes/kubernetes/blob/gh-pages/_config.yml)
+  and [_docs/index.md](https://github.com/kubernetes/kubernetes/blob/gh-pages/_docs/index.md)
+  to point to the new version. Example PR: [#16416](https://github.com/kubernetes/kubernetes/pull/16416).
+* Update [_includes/docversionselector.html](https://github.com/kubernetes/kubernetes/blob/gh-pages/_includes/docversionselector.html)
+  to make vX.Y the default version.
+* Add "Disallow: /vX.Y-1/" to existing [robots.txt](https://github.com/kubernetes/kubernetes/blob/gh-pages/robots.txt)
+  file to hide old content from web crawlers and focus SEO on new docs. Example PR:
+  [#16388](https://github.com/kubernetes/kubernetes/pull/16388).
+* Regenerate [sitemaps.xml](https://github.com/kubernetes/kubernetes/blob/gh-pages/sitemap.xml)
```

Comment:
Include the following instruction: Run the [online tool](https://www.xml-sitemaps.com/index.php?errmsg=wr&initurl=%2F) by specifying the http://kubernetes.io/ URL. Then put the new sitemap.xml file in the gh-pages branch by replacing this existing sitemap.xml file.

Need to run "resubmit sitemap" in [search console](https://www.google.com/webmasters/tools/sitemap-list?hl=en&siteUrl=http://kubernetes.io/#MAIN_TAB=0&CARD_TAB=-1) after generating sitemaps.xml.